lib/scope.js

```diff
@@ -467,27 +478,39 @@ module.exports = {
 
   normalizeTable: async function ( scope, { var_data, from_story_table=true }) {
     /* Return data derived from cucumber variable-setting data.
-    * Table must have headers. Arrays are not supported. */
+     * Table must have headers. Arrays are not supported. */
 
     // Support tables with no 'trigger' column
     let supported_table = [];
     for ( let row of var_data ) {
       // See tests > unit_tests > tables.fixtures.js > tables.old_to_current_formatting
+      let actual_var_value = row.value;
+      if (is_secret_var(row)) {
+        actual_var_value = process.env[row.flags.secret_name];
+        if ( actual_var_value === undefined ) {
+          let not_in_env_msg = `The GitHub SECRET "${ row.flags.secret_name }" doesn't exist. Check for a typo and check your workflow file.`;
+          await scope.addToReport( scope, { type: `error`, value: not_in_env_msg })
+          throw ReferenceError(`Could not find "${row.flags.secret_name }": "${not_in_env_msg}".`)
```

Comment:
This one's not super crucial either. I think some extra quotes may have snuck in, but it's hard to read this kind of stuff and keep it all in mind, so maybe I'm wrong.

```suggestion
          throw ReferenceError(`Could not find "${row.flags.secret_name }": ${not_in_env_msg}.`)
```